stock_split_picking/__manifest__.py

```diff
@@ -2,15 +2,17 @@
 # Copyright 2013-2015 Camptocamp SA - Guewen Baconnier
 # Copyright 2013-2015 Camptocamp SA - Yannick Vaucher
 # Copyright 2017 Tecnativa - Vicent Cubells
+# Copyright 2018  Versada - Andrius Preimantas
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 {
     'name': 'Split picking',
     'summary': 'Split a picking in two not transferred pickings',
-    'version': '11.0.1.0.0',
+    'version': '12.0.1.0.0',
     'category': 'Inventory',
     'author': "Camptocamp, "
               "Tecnativa, "
-              "Odoo Community Association (OCA)",
+              "Odoo Community Association (OCA), ",
+              "Versada "
```

Comment:
Generally a straight-forward migration is a contribution, it does not make you module author.

Reply:
Thanks for clarifying that. Code updated.